plugins/dfu/dfu-tool.c

```diff
@@ -997,13 +997,15 @@ main (int argc, char *argv[])
 	priv->cmd_array = g_ptr_array_new_with_free_func ((GDestroyNotify) dfu_tool_item_free);
 	dfu_tool_add (priv->cmd_array,
 		     "read",
-		     "FILENAME",
+		     /* TRANSLATORS: command argument: uppercase, spaces->dashes */
+		     _("FILENAME"),
 		     /* TRANSLATORS: command description */
 		     _("Read firmware from device into a file"),
 		     dfu_tool_read);
 	dfu_tool_add (priv->cmd_array,
 		     "read-alt",
-		     "FILENAME DEVICE-ALT-NAME|DEVICE-ALT-ID",
+		     /* TRANSLATORS: command argument: uppercase, spaces->dashes */
+		     _("FILENAME DEVICE-ALT-NAME|DEVICE-ALT-ID"),
```

Comment:
Do we need to worry about translators making these into multiple words on translation and then the number of arguments gets confused?

Reply:
I put a note in the translator comment to that effect.